src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { TransitionGroup } from 'react-transition-group';
+import { CSSTransition } from 'react-transition-group';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  loadingTodoIds: number[];
+  onRemoveTodo: (todoId: number) => Promise<void>;
+  onUpdateTodo: (todo: Todo) => Promise<void>;
```

Comment:
you sould change all functions that start with "**on**"

> exmaple:

`onUpdateTodo: (todo: Todo) => Promise<void>;`
to
`handleUpdateTodo: (todo: Todo) => Promise<void>;`

Reply:
> you sould change all functions that start with "**on**"
> 
> > exmaple:
> 
> `onUpdateTodo: (todo: Todo) => Promise<void>;` to `handleUpdateTodo: (todo: Todo) => Promise<void>;`

No, when you pass handlers as props they should start from `on`
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364